jvm/src/test/scala/org/scalacheck/GenSpecification.scala

```diff
@@ -54,10 +54,15 @@ object GenSpecification extends Properties("Gen") {
     forAll(g) { n => true }
   }
 
-  property("frequency 3") = forAll(choose(0,100000)) { n =>
+  property("frequency 3") = forAll(choose(1,100000)) { n =>
```

Comment:
Not sure why this is called "frequency 3"...should be "frequency n".

Reply:
looks like the tests on frequency are numbered 1 to 4 :)

Reply:
Ha! You are right!

Reply:
Will change to "frequency n"